riak_test/yz_pb.erl

```diff
@@ -240,7 +240,9 @@ confirm_multivalued_field(Cluster) ->
                     1 -> true;
                     0 -> false
                 end;
-            _ -> false
+            {ok, {search_results, [], _Score, 0}} ->
```

Comment:
The "catch all" case has been replaced with the tagged tuple above. Is that guaranteed to always match, or should there be an additional clause that catches anything that might sneak through?